spec/asciidocify-rendering-spec.js

```diff
@@ -0,0 +1,36 @@
+/* global describe, it, expect, asciidoctor, browser */
+describe('Load document attributes', () => {
+  const Constants = asciidoctor.browser.constants()
+  const Dom = asciidoctor.browser.dom(document)
+  const Settings = asciidoctor.browser.settings(browser, Constants)
+  const Theme = asciidoctor.browser.theme(browser, Settings, Constants)
+  const Renderer = asciidoctor.browser.renderer(browser, document, Constants, Settings, Dom, Theme)
+
+  it('should set docname correct', () => {
+    const decodeURI = document.location.href.split('/')
+    const fileName = decodeURI[decodeURI.length - 1].split('.')[0]
+    const html = Renderer.convert('= {docname}', {}).html
+    expect(html).toBe('<h1>' + fileName + '</h1>\n')
+  })
+
+  it('should set outfilesuffix correct', () => {
+    const decodeURI = document.location.href.split('/')
+    const expected = decodeURI[decodeURI.length - 1].split('.')[1]
+    const html = Renderer.convert('= {outfilesuffix}', {}).html
+    expect(html).toBe('<h1>.' + expected + '</h1>\n')
```

Comment:
I think we should use a constant here.
Actually we are lucky that in the test suite the URL is ending with `.html` (but in practice the URL will most likely end with an `.adoc`).

```suggestion
    expect(html).toBe('<h1>.html</h1>\n')
```